packages/transport-native/ios/TrezorTransportNative.m

```diff
@@ -6,4 +6,10 @@ @interface RCT_EXTERN_MODULE(TrezorTransportNative, NSObject)
                  withResolver:(RCTPromiseResolveBlock)resolve
                  withRejecter:(RCTPromiseRejectBlock)reject)
 
+// https://github.com/facebook/react-native/issues/23755#issuecomment-474235632
```

Comment:
why this? the link refers to some ImagePicker issue which seem irrelevant for us